ivy/functional/frontends/numpy/statistics/order_statistics.py

```diff
@@ -35,6 +36,13 @@ def _cpercentile(N, percent, key=lambda x: x):
     return d0 + d1
 
 
+@to_ivy_arrays_and_back

```

Comment:
As the function has out argument we will need to import `handle_numpy_out` wrapper and add it to the function